server/leafnode_test.go

```diff
@@ -335,18 +335,36 @@ func TestLeafNodeTLSRemoteWithNoCerts(t *testing.T) {
 	}
 }
 
-type captureErrorLogger struct {
+type captureLogger struct {
 	DummyLogger
-	errCh chan string
+	logCh chan string
```

Comment:
would prefer instead to have two channels, leaving the errCh as it was and add another debugCh which can be used to capture only those entries on that level

Reply:
considered that but all the tests only enable debugging when there are logs to match - so dong this didnt seem like it would improve anything

Reply:
https://github.com/nats-io/nats-server/blob/e9ee2a0901880a83741753c7de6df0f9a315afbc/server/routes_test.go#L1824-L1829